modules/broker-coordination/src/test/resources/broker.yaml

```diff
@@ -1,3 +1,3 @@
-haConfig:
+ha:
  enabled: true
```

Comment:
We might have to go through the names used when we do the broker.yaml review. Just having it as `ha` seems odd. 

Btw, Let's keep it as it is for now. :-)

Reply:
Some suggestions: clustering, coordination